firefox-ios/Client/Frontend/Settings/Main/Support/StudiesToggleSetting.swift

```diff
@@ -39,9 +40,36 @@ class StudiesToggleSetting: BoolSetting {
                 Experiments.setStudiesSetting($0)
             }
         )
-        // We make sure to set this on initialization, in case the setting is turned off
-        // in which case, we would to make sure that users are opted out of experiments
-        Experiments.setStudiesSetting(prefs.boolForKey(AppConstants.prefStudiesToggle) ?? true)
+
+        setupSettingDidChange()
+
+        let sendUsageDataPref = prefs.boolForKey(AppConstants.prefSendUsageData) ?? true
+
+        // Special Case (EXP-4780) disable studies if usage data is disabled
+        updateSetting(for: sendUsageDataPref)
+    }
+
+    private func setupSettingDidChange() {
+        self.settingDidChange = {
+            Experiments.setStudiesSetting($0)
+        }
+    }
+
+    func updateSetting(for isUsageEnabled: Bool) {
+        guard !isUsageEnabled else {
+            // Note: switch should be enabled only when telemetry usage is enabled
+            control.setSwitchTappable(to: true)
+            // We make sure to set this on initialization, in case the setting is turned off
+            // in which case, we would to make sure that users are opted out of experiments
+            Experiments.setStudiesSetting(prefs?.boolForKey(AppConstants.prefStudiesToggle) ?? true)
+            return
+        }
+
+        // Special Case (EXP-4780) disable Studies if usage data is disabled
+        control.setSwitchTappable(to: false)
+        control.toggleSwitch(to: false)
+        writeBool(control.switchView)
+        Experiments.setStudiesSetting(false)
```

Comment:
Preferred guard to ensure when someone reads the code they more or less understand why we are guarding for false case and to actually avoid just an additional curly bracket for more cleanliness. With if-else I felt doing so kinda defeats the purpose of clean code here. 

Although if you feel if-else is the way to go I don't mind changing as its such a small nit :) however you will see me as guard police in a lot of PRs instead of if-else where applicable  🙈